### PR TITLE
By adding these lines we start in the Users home folder

### PR DESCRIPTION
**Pull request by scribe**

### CODE REVIEW

**Comment by rpmoore** [comment]
I'm going to hold of on merging this until the other pr gets merged, since there are some changes in here that will conflict with the other pr.

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java [hunk 1]
```diff
@@ -87,7 +88,7 @@
     private final LoggingService loggingService;
     private final DeepStorageBrowserPresenter deepStorageBrowserPresenter;
     private final DateTimeUtils dateTimeUtils;
-    private final DataFormat local = new DataFormat("local");
+    private final DataFormat local = new DataFormat(LOCAL);
```
Comment (shabtaisharon) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java]: 👍

**Comment by shabtaisharon** [comment]
This branch is out-of-date with the base branch

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java [hunk 2]
```diff
@@ -280,8 +281,10 @@ private void createFolder() {
     }
 
     private void initProgressAndPathIndicators() {
-        final Stream<FileTreeModel> rootItems = fileTreeTableProvider.getRoot(fileRootItem);
-        localPathIndicator.setText(StringConstants.ROOT_LOCATION);
+        final String userHome = System.getProperty(StringConstants.SETTING_FILE_PATH);
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java]: Can we rename the `SETTING_FILE_PATH` constant to something that more clearly specifies that it's the user's home directory.

Comment (scribe) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java]: There we go

**Pull request merged**